enum/Dialogflow.ts

```diff
@@ -148,3 +148,14 @@ export enum Message {
 	CLOSED_BY_VISITOR = 'Closed by visitor',
 	CUSTOMER_IDLE_TIMEOUT = 'customer_idle_timeout',
 }
+
+export enum CloseChatDescription {
+	CLOSED_BY_VISITOR = 'Closed by visitor',
+	CUSTOMER_IDLE_TIMEOUT = 'Customer Timeout',
+	INVALID_TARGET_DEPARTMENT = 'Escalation failed due to invalid target department',
+	SAME_TARGET_DEPARTMENT = 'Escalation failed due to same target department',
+	LIVEAGENT_BOT_OFFLINE_OR_DISABLED = 'Escalation failed due to liveagent bot offline or disabled',
+	NETWORK_OR_APP_ERROR = 'Escalation failed due to network or app error',
+	AGENT_UNASSIGNED = 'Agent is unassigned',
+	INVALID_EVENT_DATA = 'Invalid Dialogflow event data',
```

Comment:
Will these messages end up in the widget appearing to visitors?

Can we keep the message more specific? i.e. `Chat close after the escalation failed due to invalid target department`

Reply:
No, these messages only appear when examining the omnichannel rooms in the server. 
![image](https://user-images.githubusercontent.com/18405180/193010926-080fd7c0-08b8-4d46-b97c-bef7db00c042.png)